Agents/FenlandTrajectoryAgent/example-requests/curl/gps_process_and_instantiate.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+# Copy all the following commands once, paste them into the terminal together, and then press Enter
+curl -X POST http://localhost:3840/fenland-trajectory-agent/gpstasks/fenlandtrajectoryagent/process_and_instantiate \
```

Comment:
Please update the URL, so that it is consistent with what's in the `http` files.